build/common.ps1

```diff
@@ -792,6 +828,10 @@ Function Test-CoreProjectsHelper {
     $xtests = Find-XProjects $XProjectsLocation
     $xprojs + $xtests | Restore-XProjects
 
+    # Since NuGet.Build.Tasks.Pack has "type": "build" dependencies, we need to build it directly before running its
```

Comment:
`dotnet test` will build the project as well. Why do we need to run this separately?